core/adapters/index.ts

```diff
@@ -36,7 +37,7 @@ export const requiredWarehouseProps = {
   [WarehouseType.REDSHIFT]: requiredJdbcWarehouseProps,
   [WarehouseType.SNOWFLAKE]: [
     "accountId",
-    "userName",
+    "username",
```

Comment:
whoops, this is my bad. can you please update the required JDBC properties that i changed too?

https://github.com/dataform-co/dataform/commit/86b8058497000f9926aed6ea2f0ed62f341946b2#diff-7821586da59ad263d03aff6431a570a6

Reply:
Oops - we could really do with a way to get the ACTUAL proto field names at run time and check them here as this was pretty broken.